src/server/views/dashboard/partials/details-list.njk

```diff
@@ -0,0 +1,59 @@
+{# this is a modifed copy of govuk-frontend/govuk/components/summary-list/template.njk @ 3.14.0". The original version
```

Comment:
Not sure of the route cause of this issue is but I observed a rendering issue when updating data for a lawyer.  I didn't update the emergency contact number and it was blank previously but it decided to show it had been updated something.

![Screenshot 2023-01-13 at 07 02 24](https://user-images.githubusercontent.com/79845659/212266157-f9cbf006-38bc-47e6-b0a1-6f1ceada8f88.png)

Reply:
resolved. null is treated as object in js.. 🙄. I've amended